lib/cocoapods/installer/analyzer/sandbox_analyzer.rb

```diff
@@ -223,6 +230,28 @@ def sandbox_checksum(pod)
           sandbox_manifest.checksum(pod)
         end
 
+        # @return [Dependency] The dependency with the given name stored in the sandbox.
+        #
+        # @param  [String] pod
+        #         the name of the Pod.
+        #
+        def sandbox_dependency(pod)
+          sandbox_manifest.dependencies.each { |d| return d if d.name == pod }
+          return nil
+        end
+
+        #--------------------------------------#
+
+        # @return [Dependency] The dependency with the given name from the podfile.
+        #
+        # @param  [String] pod
+        #         the name of the Pod.
+        #
+        def podfile_dependency(pod)
+          @podfile.dependencies.each { |d| return d if d.name == pod }
```

Comment:
reeplace with:

```ruby
podfile.dependencies.find { |d| d.name == pod }
```

and remove the `return nil` below. Also note `podfile` vs `@podfile` since there is an `attr_reader` at the top.